scene/gui/color_picker.cpp

```diff
@@ -855,6 +860,12 @@ ColorPicker::ColorPicker() :
 	preset_container2->add_child(bt_add_preset);
 	bt_add_preset->set_tooltip(TTR("Add current color as a preset."));
 	bt_add_preset->connect("pressed", callable_mp(this, &ColorPicker::_add_preset_pressed));
+
+	mouse_not_moving_timer = memnew(Timer);
```

Comment:
Add a comment to explain, as this isn't obvious at a glance:

```suggestion
	// Debounce color picker eyedrop preview to prevent the colorpicker from taking a long time to pick a color.
	mouse_not_moving_timer = memnew(Timer);
```